src/app/pages/ArticlePage/ReadMoreButton/index.styles.ts

```diff
@@ -38,6 +38,8 @@ export default {
       color: '#141414',
       textAlign: 'left',
       border: 'none',
+      borderBottom: '1px solid #B0B2B4', // Add the border-bottom
```

Comment:
Relative values please

Reply:
This appears to be a file you're referencing as ReadMoreButton shouldn't exist anymore.